templates/hugo/files/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+HUGOVERSION=0.55.6
+
+wget https://github.com/gohugoio/hugo/releases/download/v$HUGOVERSION/hugo_${HUGOVERSION}_Linux-64bit.tar.gz
+tar xvzf hugo_${HUGOVERSION}_Linux-64bit.tar.gz
+
+rm hugo_${HUGOVERSION}_Linux-64bit.tar.gz
+
+mv ~/themes/minimal/exampleSite/content/post/* ~/content/post/
+mv ~/themes/minimal/exampleSite/content/project/* ~/content/project/
```

Comment:
This speaks to the ongoing question of whether this should be a starting point or an example, I believe.  We don't have a definitive answer for that yet, other than "we'll want some of both".  Which this should be is unclear.

Reply:
I'm confused though.  Why are we pulling in content via submodule, AND copying it around, AND we have it included by default below, for two different themes?  I don't understand how that works.

Reply:
All this is doing is copying in `exampleSite` pages from the `minimal` theme into `app/content`. 

As far as I can tell, since this example content is not in the parent `content` directory originally, it will not be a part of the site since it is in the theme directory. This was a simple submodule that provided some example content, but not all of the ones I looked at had any. There's only one theme though.

If that didn't happen (or if we don't want to do that) there are `posts/hello-world.md` and `projects/hello-world.md` that would be the only content in those two dirs.

Reply:
The example content provided by the theme just explains how to edit when using that theme (e.g. code block syntax), which to me seemed like a starting point of "here's a working hugo template with this theme, here's where your files are, and here's how you should edit and modify them."

Reply:
If the goal is to be a starting point, then that's probably better.  Don't give people even more pages they have to remove, let them just start adding content.